src/main/java/com/hibernate/cinema/service/impl/UserServiceImpl.java

```diff
@@ -21,6 +23,7 @@ public UserServiceImpl(UserDao userDao, PasswordEncoder encoder) {
     @Override
     public User add(User user) {
         user.setPassword(encoder.encode(user.getPassword()));
+        user.setRoles(List.of(Role.of("USER")));
```

Comment:
let's do it at the AuthenticationServiceImpl
and firstly we should get from DB role by role name and then set it to the user

Reply:
do we also need to do password encoding here or in AuthenticationServiceImpl?

Reply:
no, we should do password encoding in UserService